.github/CONTRIBUTING.md

```diff
@@ -21,9 +21,9 @@ To enforce a civil & mature engagement among community members we drafted the [C
 
```

Comment:
Remove this file since it's unrelated to this PR. Also you weren't supposed to change the contents of this file without opening an issue thread.